src/main.cpp

```diff
@@ -66,7 +66,7 @@ int main(int argc, char * argv[]) {
     //QCoreApplication::setOrganizationName("Mixxx");
 
     QCoreApplication::setApplicationName(Version::applicationName());
-    QCoreApplication::setApplicationVersion(Version::version());
+    QCoreApplication::setApplicationVersion(Version::gitTag());
```

Comment:
Let's stick with version(). The git tag is only one of three version sources.

Reply:
The Git tag is the closest to what we used as simply "version" before. I have intentionally removed `Version::version` because it is ambiguous.

Reply:
This is now even more confusing, because the term git_tag() can be any string. The CMakeList.txt is responsible to make the meridian from the git tag or one of the other sources. Here we want the version().


Please keep in mind, this code here is probably read without that knowledge and misleading in that context in every other single place where used.

Reply:
The Git tag must not be any string or the whole versioning system falls apart! That is to be expected.

Reply:
Yes sure. But my argument still applies.
Please rename.

Reply:
No, there must be a clear distinction between the Git tag and `git describe`. "Version" could be anything.

Reply:
This was version() before, no need to change git_tag() is a regression for me, because it exposes an implementation detail which is only sometimes true. 

https://github.com/fakenmc/cmake-git-semver contains a possible naming convention. 
In this terms we have version() versionString() and versionFull(). 

But just keeping version would be sufficient for me.

Reply:
> it exposes an implementation detail which is only sometimes true.

This is not only sometimes true. It always comes from Git unless forcibly overriden with an incorrect value when building a prerelease snapshot outside of Git, which is an edge case. If someone messes this up, they should not be building this way and it's on them.